### ISSUE TITLE
cryptol-remote-api + :safe

### ISSUE THREAD

**Issue by weaversa** [GaloisInc/cryptol#1166]
Are there plans to add `:safe` to the Python client for `cryptol-remote-api`? It would be a nice addition to a CI framework.

**Comment by pnwamk** [comment]
Yes, I just started working on it actually. I’ll report back here once it
lands next week.

**Comment by pnwamk** [comment]
Fixed via https://github.com/GaloisInc/cryptol/commit/966b3431c19a25d978901b58dfca9d2ec15a65fd
